test/integration/agent/openflow_test.go

```diff
@@ -510,6 +523,60 @@ func checkConjunctionFlows(t *testing.T, ruleTable uint8, dropTable uint8, allow
 	}
 }
 
+func checkOVSFlowMetrics(t *testing.T, installPolicyRules bool) {
```

Comment:
this feels like a clunky test to me, and a bit hard to maintain

would the following be at all possible: simply count the flows in all the OVS tables and check that the metrics match?

Reply:
I will check if that can be done.